db/auth_on_delete_cascade.sql

```diff
@@ -0,0 +1,32 @@
+-- MySQL dump 10.13  Distrib 5.6.39, for Linux (x86_64)
+--
+-- Host: localhost    Database: atlasdb
+-- ------------------------------------------------------
+-- Server version	5.6.39
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+-- Add ON DELETE CASCADE to auth's auth_atlas_id_foreign key
+ALTER TABLE `auth` DROP FOREIGN KEY `auth_atlas_id_foreign`;
+ALTER TABLE `auth` ADD CONSTRAINT `auth_atlas_id_foreign` FOREIGN KEY (`atlas_id`) REFERENCES `atlas`(`id`) ON DELETE CASCADE;
```

Comment:
@bmamlin I think we need an upgrade script here?

Reply:
Yes. You could put the conditional upgrade into a file (like `auth.v3.sql`) that could be pulled into the upgrade script for #90. Let's get this PR and #95 done, get #90 updated & merge, deploy to staging, and if it looks good, we'll deploy to production! Once it's in production, we can incorporate all changes in the update script into the schema script and remove them from the update script.

Reply:
@bmamlin Do we need a conditional upgrade here? The output is the same even if ON DELETE CASCADE is present in the constraint. If we need a conditional upgrade, should I make the change in this PR or in #90?